plugins/wazo_fanvil/common/common.py

```diff
@@ -523,7 +537,9 @@ def _add_phonebook_url_v2(self, raw_config: dict[str, Any]) -> None:
             fixed_url = raw_config['XX_xivo_phonebook_url'].replace('&', '&amp;')
 
             raw_config['XX_wazo_phonebook_url_v2'] = fixed_url
-            raw_config['XX_xivo_phonebook_url'] = original_phonebook_url
+            raw_config['XX_xivo_phonebook_url'] = original_phonebook_url.replace(
```

Comment:
This method is only supposed to modify `XX_wazo_phonebook_url_v2`. The addition of a side effect on `XX_xivo_phonebook_url` is no good. I suggest moving that change to the other method, `_add_phonebook_url`.